src/main/java/reposense/report/SummaryReportJson.java

```diff
@@ -8,11 +8,11 @@
  * Represents the structure of summary.json file in reposense-report folder.
  */
 public class SummaryReportJson {
-    private final String dashboardGeneratedTime;
+    private final String reportGeneratedTime;
     private final List<RepoConfiguration> repos;
 
-    public SummaryReportJson(List<RepoConfiguration> repos, String dashboardGeneratedTime) {
+    public SummaryReportJson(List<RepoConfiguration> repos, String reportGeneratedTime) {
         this.repos = repos;
-        this.dashboardGeneratedTime = dashboardGeneratedTime;
+        this.reportGeneratedTime = reportGeneratedTime;
```

Comment:
Doing this will cause side effects. Now, the generated report can not show the time correctly.
You can verify it through the netlify deploy.
![image](https://user-images.githubusercontent.com/35621726/53621180-9fe20f80-3c30-11e9-861b-78850210e44c.png)
To solve this, rename the variable [here](https://github.com/reposense/RepoSense/blob/master/frontend/src/static/js/api.js#L35) as well.

Reply:
Done 👍